### DISCUSSION TITLE
Toggle on/off hit animations in osu standard

### DISCUSSION THREAD

**Discussion by Joanthenoob** [ppy/osu#17583] [Ideas] [answered]
It would be really nice if there was a way to turn off hit animations, i.e. when a circle is clicked it disappears instead of "pop" out. People already do it in stable by adding hitcircles to the defaults but that prevents the use of combo colors and 2 digit comboing doesn't really work so i feel having hit animations as a toggle in lazer.

**Comment by integerrr** [comment, marked as answer]
#15123
